java/src/main/java/com/google/security/binexport/BinExport2Builder.java

```diff
@@ -407,6 +411,22 @@ private void buildCallGraph() throws CancelledException {
     }
   }
 
+  private void buildDataReferences(Map<Long, Integer> instructionIndices ) {
```

Comment:
Can you please reformat this file to conform to the Google Java style guide?
Note that you can also send me a Piper CL directly (this lives in `//third_party/zynamics/binexport/java`)

Reply:
Good catch, I've updated the formatting. Would you prefer future contributions come directly through Piper CLs?

Reply:
> Good catch, I've updated the formatting. Would you prefer future contributions come directly through Piper CLs?

Yes, actually. Not because I don't want to discuss in the open, but because Critique is just much nicer for commenting and reviews.